frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -116,7 +116,6 @@ function _BaseBucketPickerPopover({
       opened={isOpened}
       position="right"
       onClose={handlePopoverClose}
-      withinPortal={false}
```

Comment:
@romeovs try https://github.com/metabase/metabase/blob/5a1d6774441f865812dc05fc39db1980493713da/frontend/src/metabase/visualizations/components/settings/ChartSettingSelect.jsx#L69

Reply:
This is already set on this component [in this line](https://github.com/metabase/metabase/pull/58170/files#diff-304d293d38124406ea9adcf57767c473070b68427d2e5036730b95334a157d19R120), but that doesn't help.

I think it's a Safari bug wrt. z-index and overflow but I'm struggling to track down where the overflow if set.